app/views.py

```diff
@@ -101,6 +135,7 @@ def createclass():
         return render_template('error404.html'), 404
     classForm = CreateName(prefix='class')
     quizForm = CreateName(prefix='quiz')
```

Comment:
```suggestion
    classForm = NameForm(prefix='class')
    quizForm = NameForm(prefix='quiz')
```
To reflect the changes in `forms.py`.